<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsWindow</class>
 <widget class="QWidget" name="settingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="AutoReplace">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Автоматическая замена файлов при перемещении</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="levelCount">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="levelCountLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество контролируемых путей вверх</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
